execution/gethexec/executionengine.go

```diff
@@ -654,17 +725,62 @@ func (s *ExecutionEngine) getL1PricingSurplus() (int64, error) {
 	return surplus.Int64(), nil
 }
 
-func (s *ExecutionEngine) cacheL1PriceDataOfMsg(num arbutil.MessageIndex, receipts types.Receipts, block *types.Block) {
+func (s *ExecutionEngine) cacheL1PriceDataOfMsg(seqNum arbutil.MessageIndex, receipts types.Receipts, block *types.Block, blockBuiltUsingDelayedMessage bool) {
 	var gasUsedForL1 uint64
-	for i := 1; i < len(receipts); i++ {
```

Comment:
I am not completely sure why we don't consider the first receipt here.
I added a commit fixing this, but flagging here as a comment so the reviewer can double check this.

Reply:
OK, thanks!